package.json

```diff
@@ -280,7 +279,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn check-commit && yarn documentation && pretty-quick --check"
+      "pre-push": "yarn documentation && pretty-quick --check"
```

Comment:
I'd still run check-commit. I think sometimes it's useful to not squash commits. And I think it's good to get the feedback when you push.

Reply:
I've reverted back to including the `check-commit` here.

As a matter of personal preference, I find the pre-push hook cumbersome.

How would you feel about some warning message pre-commit to warn users if there's a failing commit message. Something like:

```
WARNING: The commit message you provided does not pass our formatting rules.

If you're planning to use the 'Squash & Merge' strategy on Github to flatten your commits, please be sure to create a properly formatted commit message for the PR's title. Our Github Actions will validate the PR title for you.

If you do not intend to 'Squash & Merge', please rename this commit (https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message).

EXPECTED FORMAT: <TYPE>(<SCOPE>): <DESCRIPTION>
+ SCOPE is required
+ Valid SCOPES: 'angular', 'core', 'nxdev', 'nextjs', 'gatsby', 'nest', 'node', 'express', 'nx-plugin', 'react', 'web', 'linter', 'storybook', 'testing', 'repo', 'misc', 'devkit'
+ Valid TYPES: 'feat', 'fix', 'docs', 'cleanup', 'chore'
```